src/track/beatfactory.cpp

```diff
@@ -34,8 +34,8 @@ mixxx::BeatsPointer BeatFactory::loadBeatsFromByteArray(
 mixxx::BeatsPointer BeatFactory::makeBeatGrid(
         mixxx::audio::SampleRate sampleRate,
         double dBpm,
-        double dFirstBeatSample) {
-    return mixxx::BeatGrid::makeBeatGrid(sampleRate, QString(), dBpm, dFirstBeatSample);
+        mixxx::audio::FramePos firstBeatFramePos) {
```

Comment:
Please double check that this is correct. The old variable name was `dFirstBeatSample`, but tests still pass so I suppose it was just a super confusing misnomer. The beatmap certainly uses frames (not samples) for input.

Reply:
The old name was correct for makeBeatGrid() as samples.
You have changed it here and you should have catched all occurrences.
Just double checked it.